web/README.md

```diff
@@ -64,3 +64,73 @@ the former command executes.
    tab.
 4. In the Dev console, you can set a breakpoint in your test and refresh the
    page to debug
+
+### Approximate Overall Design
+
+```mermaid
+graph TD;
```

Comment:
While this code block below doesn't render well offline, or in reviews, it displays beautifully when directly viewing it within GitHub.

We should be able to prerender it for offline viewing using https://github.com/mermaid-js/mermaid-cli if desired; that does mean adding more dev-dependencies, though.